src/System.Runtime.Numerics/tests/BigInteger/add.cs

```diff
@@ -1,160 +0,0 @@
-                    byteArray2 = GetRandomByteArray(s_random, 2);
-                    VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-
-                    byteArray1 = GetRandomByteArray(s_random, 2);
-                    byteArray2 = GetRandomByteArray(s_random);
-                    VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-                }
-                catch (IndexOutOfRangeException)
-                {
-                    Console.WriteLine("Array1: " + Print(byteArray1));
-                    Console.WriteLine("Array2: " + Print(byteArray2));
-                    throw;
-                }
-            }
-
-            // Add Method - One large BigInteger and zero
-            for (int i = 0; i < s_samples; i++)
-            {
-                byteArray1 = GetRandomByteArray(s_random);
-                byteArray2 = new byte[] { 0 };
-                VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-
-                byteArray1 = new byte[] { 0 };
-                byteArray2 = GetRandomByteArray(s_random);
-                VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-            }
-
-            // Add Method - One small BigInteger and zero
-            for (int i = 0; i < s_samples; i++)
-            {
-                byteArray1 = GetRandomByteArray(s_random, 2);
-                byteArray2 = new byte[] { 0 };
-                VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-
-                byteArray1 = new byte[] { 0 };
-                byteArray2 = GetRandomByteArray(s_random, 2);
-                VerifyAdditionString(Print(byteArray1) + Print(byteArray2) + "bAdd");
-            }
-
-            // 32 bit boundary n1=0 n2=0 c=0
-            VerifyAdditionString("0 0 bAdd");
-
-            // 32 bit boundary n1=0 n2=0 c=1
-            VerifyAdditionString((Math.Pow(2, 31) + Math.Pow(2, 30)) + " " + (Math.Pow(2, 31) + Math.Pow(2, 30)) + " bAdd");
-
-            // 32 bit boundary n1=0 n2=1 c=0
-            VerifyAdditionString("0" + " " + Math.Pow(2, 32) + " bAdd");
-
-            // 32 bit boundary n1=0 n2=1 c=1
-            VerifyAdditionString(Math.Pow(2, 31) + " " + (Math.Pow(2, 32) + Math.Pow(2, 31)) + " bAdd");
-
-            // 32 bit boundary n1=1 n2=0 c=0
-            VerifyAdditionString(Math.Pow(2, 32) + " " + "0" + " bAdd");
-
-            // 32 bit boundary n1=1 n2=0 c=1
-            VerifyAdditionString((Math.Pow(2, 32) + Math.Pow(2, 31)) + " " + Math.Pow(2, 31) + " bAdd");
-
-            // 32 bit boundary n1=0 n2=1 c=0
-            VerifyAdditionString(Math.Pow(2, 32) + " " + Math.Pow(2, 32) + " bAdd");
-
-            // 32 bit boundary n1=0 n2=1 c=1
-            VerifyAdditionString((Math.Pow(2, 32) + Math.Pow(2, 31)) + " " + (Math.Pow(2, 32) + Math.Pow(2, 31)) + " bAdd");
-
-            // Identity (x+y)+z == (y+z)+x
-            VerifyIdentityString(
```

Comment:
These Identity tests are omitted, as we now check the following:
```
Assert.Equal(expected, bigInteger1 + bigInteger2);
Assert.Equal(expected, bigInteger2 + bigInteger1);

Assert.Equal(expected, BigInteger.Add(bigInteger1, bigInteger2));
Assert.Equal(expected, BigInteger.Add(bigInteger2, bigInteger1));
```
This is a better way of making sure `(x+y) === (y+x)`